{ 'stand_up_complete': { 'left_arm': [ { 'duration': 1.0,
                                         'name': u'stand_up_0',
                                         'positions': [ -1.5878,
                                                        1.2968,
                                                        -0.0002,
                                                        -0.0001,
                                                        -0.7176,
                                                        -0.2511,
                                                        0.0001],
                                         'starttime': 0.0},
                                       { 'duration': 1.44,
                                         'name': u'stand_up_1',
                                         'positions': [ -2.0902,
                                                        0.611,
                                                        0.0024,
                                                        -0.0012,
                                                        -1.2463,
                                                        0.0029,
                                                        0.0023],
                                         'starttime': 1.002},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_1_2',
                                         'positions': [ -2.2876,
                                                        0.4934,
                                                        -0.0042,
                                                        -0.0285,
                                                        -1.2588,
                                                        -0.0098,
                                                        0.0],
                                         'starttime': 2.448},
                                       { 'duration': 1.808,
                                         'name': u'stand_up_2',
                                         'positions': [ -0.268,
                                                        -1.6445,
                                                        -0.0204,
                                                        -0.0428,
                                                        -1.2599,
                                                        -0.017,
                                                        0.0001],
                                         'starttime': 4.531},
                                       { 'duration': 1.636,
                                         'name': u'stand_up_3',
                                         'positions': [ -0.3277,
                                                        -2.4852,
                                                        -0.1177,
                                                        -0.2064,
                                                        0.0692,
                                                        -0.0233,
                                                        0.1897],
                                         'starttime': 6.349},
                                       { 'duration': 2.001,
                                         'name': u'stand_up_4',
                                         'positions': [ -1.7973,
                                                        -1.0732,
                                                        -0.1109,
                                                        -0.206,
                                                        0.0616,
                                                        -0.02,
                                                        0.19],
                                         'starttime': 7.985},
                                       { 'duration': 2.245,
                                         'name': u'stand_up_5',
                                         'positions': [ -0.172,
                                                        0.0622,
                                                        -0.1055,
                                                        -0.2186,
                                                        0.0601,
                                                        -0.02,
                                                        0.19],
                                         'starttime': 11.22}],
                         'left_leg': [ { 'duration': 1.0,
                                         'name': u'stand_up_0',
                                         'positions': [ -0.0153,
                                                        0.0076,
                                                        -0.3478,
                                                        0.7871,
                                                        0.8288,
                                                        -0.0573],
                                         'starttime': 0.0},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_1',
                                         'positions': [ -0.0099,
                                                        -0.0042,
                                                        0.0158,
                                                        0.5839,
                                                        1.571,
                                                        -0.0598],
                                         'starttime': 3.498},
                                       { 'duration': 1.812,
                                         'name': u'stand_up_2',
                                         'positions': [ -0.0104,
                                                        -0.0104,
                                                        -1.3423,
                                                        2.1665,
                                                        1.57,
                                                        -0.06],
                                         'starttime': 4.532},
                                       { 'duration': 1.636,
                                         'name': u'stand_up_3',
                                         'positions': [ -0.0074,
                                                        -0.0073,
                                                        -1.5555,
                                                        3.1374,
                                                        1.8221,
                                                        -0.0594],
                                         'starttime': 6.362},
                                       { 'duration': 2.859,
                                         'name': u'stand_up_5',
                                         'positions': [ -0.027,
                                                        -0.0151,
                                                        -1.5847,
                                                        3.1403,
                                                        1.5875,
                                                        -0.035],
                                         'starttime': 11.222},
                                       { 'duration': 8.304,
                                         'name': u'stand_up_7',
                                         'positions': [ -0.0405,
                                                        -0.0199,
                                                        -0.3577,
                                                        0.6864,
                                                        0.3657,
                                                        -0.0202],
                                         'starttime': 14.081}],
                         'head': [ { 'duration': 1.0,
                                     'name': u'stand_up_0',
                                     'positions': [0.0001, -0.9375],
                                     'starttime': 0.0}],
                         'right_arm': [ { 'duration': 1.0,
                                          'name': u'stand_up_0',
                                          'positions': [ 1.5878,
                                                         -1.2968,
                                                         0.0002,
                                                         0.0001,
                                                         0.7176,
                                                         0.2511,
                                                         -0.0001],
                                          'starttime': 0.0},
                                        { 'duration': 1.44,
                                          'name': u'stand_up_1',
                                          'positions': [ 2.0902,
                                                         -0.611,
                                                         -0.0024,
                                                         0.0012,
                                                         1.2463,
                                                         -0.0029,
                                                         -0.0023],
                                          'starttime': 1.002},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_1_2',
                                          'positions': [ 2.2876,
                                                         -0.4934,
                                                         0.0042,
                                                         0.0285,
                                                         1.2588,
                                                         0.0098,
                                                         -0.0],
                                          'starttime': 2.448},
                                        { 'duration': 1.808,
                                          'name': u'stand_up_2',
                                          'positions': [ 0.268,
                                                         1.6445,
                                                         0.0204,
                                                         0.0428,
                                                         1.2599,
                                                         0.017,
                                                         -0.0001],
                                          'starttime': 4.531},
                                        { 'duration': 1.636,
                                          'name': u'stand_up_3',
                                          'positions': [ 0.3277,
                                                         2.4852,
                                                         0.1177,
                                                         0.2064,
                                                         -0.0692,
                                                         0.0233,
                                                         -0.1897],
                                          'starttime': 6.349},
                                        { 'duration': 2.001,
                                          'name': u'stand_up_4',
                                          'positions': [ 1.7973,
                                                         1.0732,
                                                         0.1109,
                                                         0.206,
                                                         -0.0616,
                                                         0.02,
                                                         -0.19],
                                          'starttime': 7.985},
                                        { 'duration': 2.245,
                                          'name': u'stand_up_5',
                                          'positions': [ 0.172,
                                                         -0.0622,
                                                         0.1055,
                                                         0.2186,
                                                         -0.0601,
                                                         0.02,
                                                         -0.19],
                                          'starttime': 11.22}],
                         'right_leg': [ { 'duration': 1.0,
                                          'name': u'stand_up_0',
                                          'positions': [ 0.0153,
                                                         -0.0076,
                                                         0.3478,
                                                         -0.7871,
                                                         -0.8288,
                                                         0.0573],
                                          'starttime': 0.0},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_1',
                                          'positions': [ 0.0099,
                                                         0.0042,
                                                         -0.0158,
                                                         -0.5839,
                                                         -1.571,
                                                         0.0598],
                                          'starttime': 3.498},
                                        { 'duration': 1.812,
                                          'name': u'stand_up_2',
                                          'positions': [ 0.0104,
                                                         0.0104,
                                                         1.3423,
                                                         -2.1665,
                                                         -1.57,
                                                         0.06],
                                          'starttime': 4.532},
                                        { 'duration': 1.636,
                                          'name': u'stand_up_3',
                                          'positions': [ 0.0074,
                                                         0.0073,
                                                         1.5555,
                                                         -3.1374,
                                                         -1.8221,
                                                         0.0594],
                                          'starttime': 6.362},
                                        { 'duration': 2.859,
                                          'name': u'stand_up_5',
                                          'positions': [ 0.027,
                                                         0.0151,
                                                         1.5847,
                                                         -3.1403,
                                                         -1.5875,
                                                         0.035],
                                          'starttime': 11.222},
                                        { 'duration': 8.108,
                                          'name': u'stand_up_7',
                                          'positions': [ 0.0405,
                                                         0.0199,
                                                         0.3577,
                                                         -0.6864,
                                                         -0.3657,
                                                         0.0202],
                                          'starttime': 14.081}],
                         'torso': [ { 'duration': 1.0,
                                      'name': u'stand_up_0',
                                      'positions': [0.0001, 0.0107],
                                      'starttime': 0.0},
                                    { 'duration': 1.84,
                                      'name': u'stand_up_2',
                                      'positions': [0.0016, 0.532],
                                      'starttime': 4.544},
                                    { 'duration': 1.636,
                                      'name': u'stand_up_3',
                                      'positions': [-0.0017, 1.5741],
                                      'starttime': 6.401},
                                    { 'duration': 3.145,
                                      'name': u'stand_up_4',
                                      'positions': [-0.007, -0.0831],
                                      'starttime': 8.037}]}}
